### PR TITLE
Update TF-M version in CI FIH test

### PR DESCRIPTION
**Pull request by tamasban**
Update from `TF-Mv1.4.0` to `TF-Mv1.7.0`.

### CODE REVIEW

File: ci/fih_test_docker/fi_tester_gdb.sh [hunk 1]
```diff
@@ -40,6 +40,13 @@ function skip_instruction {
 
     cat >commands.gdb <<EOF
 target remote localhost: 1234
+file $IMAGE_DIR/bl2.axf
+b boot_go_for_image_id
```
Comment (RcColes) [inline ci/fih_test_docker/fi_tester_gdb.sh:44]: Nit: I think this should be able to be a `b boot_go_for_image_id if image_id != 0`, which makes the logic slightly nicer. Could well be wrong though, I hardly ever use conditional breakpoints these days.

File: ci/fih_test_docker/execute_test.sh [hunk 2]
```diff
@@ -34,6 +34,9 @@ else
 fi
 
 # build TF-M with MCUBoot
+
+export GIT_SSL_NO_VERIFY=1
```
Comment (RcColes) [inline ci/fih_test_docker/execute_test.sh:38]: What's the purpose of this? I'd be concerned about disabling SSL

Comment (tamasban) [reply ci/fih_test_docker/execute_test.sh:38]: When I tried to run the scripts in my local environment I faced the issue below during TF-M build. This is a workaround for that:

```
Cloning into 'tfm_test_repo-src'...
fatal: unable to access 'https://git.trustedfirmware.org/TF-M/tf-m-tests.git/': server certificate verification failed. CAfile: none CRLfile: none

Interestingly I can pull the code from the same machine, but the scripts run in a docker container. Might this cause the issue?

Comment (tamasban) [reply ci/fih_test_docker/execute_test.sh]: Removed since it not an issue in GitHub CI.

**Review by RcColes — APPROVED** [review]

**Review by nvlsianpu — APPROVED** [review]

**Pull request merged**